### PR TITLE
Add $GRDN

### PR DESCRIPTION
**Pull request by datmax**

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Thank you. Do you have an SVG of the token logo? There only a PNG in the Chain Registry, but an SVG will look sharper and take less size.

**Comment by datmax** [comment]
> Thank you. Do you have an SVG of the token logo? There only a PNG in the Chain Registry, but an SVG will look sharper and take less size.
> 
> Nvm, I don't even see it in the Chain Registry. Please register it to the Chain Registry.

Pity me, sorry for the inconvenience. Just made a PR to add the token on the chain registry. Thanks for the input. We only have a png version of the logo at the moment and our artist is not available, so we'll have to stick with the png for now.

**Comment by datmax** [comment]
PR on chain registry got approved.

File: osmosis-1/osmosis.zone.json [hunk 1]
```diff
@@ -1627,6 +1627,16 @@
...
+    {
+      "chain_name": "juno",
+      "base_denom": "cw20:juno1xekkh27punj0uxruv3gvuydyt856fax0nu750xns99t2qcxp7xmsqwhfma",
+      "pools": {
+        "OSMO": 959
+      },
+      "osmosis_main": false,
+      "osmosis_frontier": true,
+      "osmosis_info": true
+    },
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone.json:1640]: oh, please remove this comma

**Comment by JeremyParish69** [comment]
Could you please validate deposit and withdrawal with this staging link?
https://osmosis-frontier-git-jp69-add-grdn-osmo-labs.vercel.app/

**Comment by datmax** [comment]
Comma removed and validated deposit/withdraw. Sorry for the trouble.

**Comment by datmax** [comment]
PR got approved but seems like we're not present on the assetlist. Can this please get reviewed?

**Pull request merged**